emrun.py

```diff
@@ -34,14 +34,14 @@
 from operator import itemgetter
```

Comment:
This file is explicitly designed to run on both python2 and python3.  We should probably specify that explicitly in  the comment at the top.